providers/flagd/README.md

```diff
@@ -61,13 +61,14 @@ variables.
 Given below are the supported configurations:
 
 | Option name           | Environment variable name      | Type & Values          | Default   | Compatible resolver |
-| --------------------- | ------------------------------ | ---------------------- | --------- | ------------------- |
+|-----------------------|--------------------------------|------------------------|-----------|---------------------|
 | host                  | FLAGD_HOST                     | String                 | localhost | RPC & in-process    |
 | port                  | FLAGD_PORT                     | int                    | 8013      | RPC & in-process    |
 | tls                   | FLAGD_TLS                      | boolean                | false     | RPC & in-process    |
 | socketPath            | FLAGD_SOCKET_PATH              | String                 | null      | RPC & in-process    |
 | certPath              | FLAGD_SERVER_CERT_PATH         | String                 | null      | RPC & in-process    |
 | deadline              | FLAGD_DEADLINE_MS              | int                    | 500       | RPC & in-process    |
+| selector              | FLAGD_SELECTOR                 | String                 | null      | in-process          |
```

Comment:
Please make sure this is also included in the flagd documentation so that it's included in other in-process evaluators.

Reply:
Actually this is already defined in the spec - https://github.com/open-feature/flagd/blob/main/docs/other_resources/in-process-providers/specification.md#sync-source 

`FLAGD_SOURCE_SELECTOR: Optional selector for the feature flag configuration of interest. This is used as a selector for the flagd-proxie's sync API to identify a flag configuration within a collection of feature flag configurations.
`

I will update the PR to have the same naming convention

Reply:
Updated - https://github.com/open-feature/java-sdk-contrib/pull/451/files#diff-04582b0607c583d36a31ea656e9229beceab5f2abc3139710831765da374ce89R71